src/vcpkg/paragraphs.cpp

```diff
@@ -441,34 +441,34 @@ namespace vcpkg::Paragraphs
                                   std::string{}};
         }
 
-        if (fs.exists(port_location.port_directory, IgnoreErrors{}))
-        {
-            return PortLoadResult{LocalizedString::from_raw(port_location.port_directory)
-                                      .append_raw(": ")
-                                      .append_raw(ErrorPrefix)
-                                      .append(msgPortMissingManifest2, msg::package_name = port_name),
-                                  std::string{}};
-        }
-
-        return PortLoadResult{LocalizedString::from_raw(port_location.port_directory)
-                                  .append_raw(": ")
-                                  .append_raw(ErrorPrefix)
-                                  .append(msgPortDoesNotExist, msg::package_name = port_name),
-                              std::string{}};
+        return PortLoadResult{SourceControlFileAndLocation{}, std::string{}};
     }
 
     PortLoadResult try_load_port_required(const ReadOnlyFilesystem& fs,
                                           StringView port_name,
                                           const PortLocation& port_location)
     {
-        auto load_result = try_load_port(fs, port_name, port_location);
+        auto load_result = try_load_port(fs, port_location);
```

Comment:
If this is the only caller, should try_load_port be inlined?

Reply:
I intend to add other callers :)

Reply:
(Specifically, how I discovered this situation: overlay-ports want to try-load-port and it's OK if there's no port there: because it might be a directory containing overlays rather than the overlay itself)